third-party/CMakeLists.txt

```diff
@@ -81,6 +81,15 @@ if(CMAKE_GENERATOR MATCHES "Makefiles")
   set(MAKE_PRG "$(MAKE)")
 endif()
 
+if(MINGW AND CMAKE_GENERATOR MATCHES "Ninja")
+  find_program(MAKE_PRG NAMES mingw32-make)
+  if(NOT MAKE_PRG)
+    message(FATAL_ERROR "GNU Make for mingw32 is required to build the dependecies.")
+  else()
+    message(STATUS "Found GNU Make for mingw32 at ${MAKE_PRG}")
+  endif()
+endif()
```

Comment:
so mingw32-make is required even when building with MSVC? Does the [wiki](https://github.com/neovim/neovim/wiki/Building-Neovim#windows--msvc) instructions need to mention this?